examples/portfolio/src/components/Nav.astro

```diff
@@ -60,7 +60,7 @@ const iconLinks: { label: string; href: string; icon: keyof typeof iconPaths }[]
 			}
 		</ul>
 	</noscript>
-	<noscript style="display: contents;">
+	<noscript style="menu-content">
```

Comment:
The value for style seems invalid, does it do anything?

Reply:
You're right, and the `<noscript>` tag doesn't need the style attribute (it's invalid and not doing anything), and the other one doesn't need a class (it doesn't match a rule anyway). They might be causes of confusion for users building on top of this example.

Reply:
So, should I remove both style selectors from both <noscript/> tags?

Reply:
Yeah, that sounds good to me.